reinforcement_learning/doc/mainpage.dox

```diff
@@ -0,0 +1,34 @@
+/*! \mainpage Introduction
+
+Reforcement learning deals with training data that is not labeled.  Either it's not available or not practical to obtain.  Instead a learner is trained to pick the 
```

Comment:
How about we replace this paragraph with the following (feel free to swizzle it a bit more): 

Enterprises are constantly faced with decisions that require picking from a set of actions based on contextual
information. Real world reinforcement-based techniques are effective tools in aiding decision making; they rely on free interaction data to "predict" and "learn". This library implements an interface for reinforcement-based prediction based on contextual bandits.

Reply:
Can't think of how to say it better.  Done.